src/MPI.jl

```diff
@@ -58,6 +58,23 @@ end
 include("consts/consts.jl")
 using .Consts
 
+module API
+    using ..Consts
+    import ..libmpi
+
+    for name in filter(n -> startswith(string(n), "MPI_"), names(Consts; all = true))
+        @eval $name = Consts.$name  # signatures need types
+    end
+
+    struct MPIError <: Exception
+        code::Cint
+    end
+    macro mpichk(expr)
```

Comment:
I guess `@mpichk` in the top-module should be a const alias of this one?

Reply:
We shouldn't need it in the top-level now, just move it in here

Reply:
Yes, solved.